app/controllers/api/v1/code_schools_controller.rb

```diff
@@ -42,10 +42,19 @@ def destroy
         end
       end
 
+      def get_moocs
```

Comment:
Can we change the name of this action to be in keeping the `scope` like nature of its purpose.  And just call it `moocs`.

This way the url endpoint can also read like a scope on the model, i.e. `api/v1/code_schools/moocs`.

You'll want to adjust the route, accordingly.